File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.java
class Solution {
    public int maxCoins(int[] piles) {
        Arrays.sort(piles);
        int z=(piles.length)/3;
        int ans=0;
        for(int i=piles.length-2;i>=piles.length-z*2;i-=2){
            ans+=piles[i];
        }
        return ans;
    }
}

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort()
        z=len(piles)//3
        x=-2
        ans=0
        while z!=0:
            ans+=piles[x]
            x-=2
            z-=1
        return ans